src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    private OperationStrategyImpl operationStrategy;
+    private OperationHandler balanceHandler;
+    private OperationHandler purchaseHandler;
+    private OperationHandler supplyHandler;
+
+    @BeforeEach
+    void setUp() {
+        balanceHandler = (fruit, quantity) -> {};
+        purchaseHandler = (fruit, quantity) -> {};
+        supplyHandler = (fruit, quantity) -> {};
```

Comment:
```suggestion
        balanceHandler = (fruit, quantity) -> {};
        purchaseHandler = (fruit, quantity) -> {};
        supplyHandler = (fruit, quantity) -> {};
```
Let’s initialize handlers. Return handler missed

Reply:
Not fixed. If you have any questions, let’s  discuss it in the chat